app/lib/actions/katello/orphan_cleanup/remove_orphaned_content_units.rb

```diff
@@ -0,0 +1,32 @@
+module Actions
+  module Katello
+    module OrphanCleanup
+      class RemoveOrphanedContentUnits < Actions::Base
+        def plan(options = {})
+          plan_self(id: options[:repo_id], destroy_all: options[:destroy_all])
+        end
+
+        def run
+          content_types_to_index = []
```

Comment:
`content_types_to_index` makes it sound a bit like the types will be indexed instead of checked for orphans.